### ISSUE TITLE
Viewing examples on dashboard

### ISSUE THREAD

**Issue by andrew-jones** [instana/go-sensor#165]
Hi,

I'm having issues getting trace metadata into Instana, or finding it on the dashboard. 
 I've tried the go-sensor examples,

https://github.com/instana/go-sensor/blob/master/example/ot-simple/simple.go
https://github.com/instana/go-sensor/blob/master/example/rpc/rpc.go

I can get the calls coming through to the Instana dashboard and the services show up as I would expect. There's just no detail on the traces. Span names, any tags, log fields etc don't seem to be anywhere. 

For ot-simple:
* The name of the endpoint is "GET /golang" for ot-simple (where does this come from?)
* Where is the span name displayed?  "asteroid" in this case
* Where are the tags?  (Component, Url, Method,)
* Where are the log fields?

For rpc:
* The endpoint name is "Internal trigger"
* Where are the endpoints parentService.myCoolMethod and childService.anotherCoolMethod?

thanks!

**Comment by andrew-jones** [comment]
![Screenshot from 2020-10-15 16-02-04](https://user-images.githubusercontent.com/1200734/96192259-63902e00-0f91-11eb-8ad6-998a71b8110e.png)

**Comment by andrew-jones** [comment]
![Screenshot from 2020-10-15 16-02-04](https://user-images.githubusercontent.com/1200734/96192315-80c4fc80-0f91-11eb-8f08-6860c9da01ac.png)

**Comment by andrewslotin** [comment]
Hey @andrew-jones!

Thanks for bringing this up. The tracer has evolved since these examples were added and the most up-to-date examples can be found in the package [godoc](https://pkg.go.dev/github.com/instana/go-sensor). Here is an [example of an HTTP server instrumentation](https://pkg.go.dev/github.com/instana/go-sensor#example-package-TracingHandlerFunc) and [this one](https://pkg.go.dev/github.com/instana/go-sensor/instrumentation/instagrpc#example-package) demonstrates how to instrument a gRPC server.

To answer your questions, the span attributes and logs can be found in the span details pane, this is the one if you click on the call name ("Internal trigger" in your case) and pick the call to inspect. These tags can also be used to filter calls, e.g. to see the HTTP requests that ended up with 500.

> The endpoint name is "Internal trigger"

As I mentioned above, the examples in the `examples/` folder may be [outdated](https://github.com/instana/go-sensor/blob/master/example/README.md). In this case the call does not start with an entry span (set by the `span.kind` annotation). This is a common case for operations that are started by an internal trigger, such as timer, an internal event, etc., so we're correcting this while processing the call by inserting an entry span with operation name "Internal trigger".

**Comment by andrew-jones** [comment]
Hi @andrewslotin,

Thanks alot for the reply, I've gone through the instagrpc example and modified for my case and its working nicely!  I've tested the go-sensor and database/sql integration also.

I'm having an issue setting the error message displayed on the dashboard after an rpc error.  I've set "rpc.error" and it comes up in the analyse calls screens, but the "Applications"->"Services"->"Error Messages" tab on an endpoint shows something generic ("Erroneous call without error message").

Cheers!
